libraries/SITL/SITL.cpp

```diff
@@ -453,6 +454,7 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     AP_GROUPINFO("INIT_ALT_OFS",  47, SIM,  gps_init_alt_ofs, 0),
 
     AP_GROUPINFO("GPS_LOG_NUM",   48, SIM,  gps_log_num, 0),
+    AP_GROUPINFO("GPS2_JAM",      49, SIM,  gps_jam[1], 0),
```

Comment:
we accept parametr descriptions for SITL variables so a little set of param descriptions above this might be helpful in using it.

Reply:
Busted!